app-creator/src/widgets/dropzone-widget/dropzone-widget.ts

```diff
@@ -108,12 +102,16 @@ export class Dropzone extends LitElement {
       } else {
         container.insertBefore(widgetWrapper, nextElement);
       }
+      // set the global reordering state to true so we know that we don't increment the current widget id
```

Comment:
Could we break this part of function into two functions - reordering an editing, so it's clearer which case we are handling?